### DISCUSSION TITLE
How could I output the data that the resulting pressure field can be visualized in Paraview>

### DISCUSSION THREAD

**Discussion by weishiyu1** [PMEAL/OpenPNM#2280] [Q&A]
when I run StokesFlow Algorithm, how could I output the data that the resulting pressure field can be visualized in Paraview, `https://github.com/PMEAL/OpenPNM/blob/dev/examples/notebooks/io/loading_statoil_finding_permeability.ipynb` have output results, but no related code is mentioned. Thank you in advance.

**Comment by jgostick** [comment]
This is the second time a discussion has gone unanswered for too long.  I"m sorry for missing this.  We accidentally deleted the instructions for visualizing in paraview, but have reposted them here:  https://github.com/PMEAL/OpenPNM/blob/readd_paraview/examples/notebooks/io/view_vtp_in_paraview.md.  Hope that helps a bit.  Note that you'll want to include your ``phase`` object in the list of phases that you pass when you call the ``vtk`` io function.  I should also add that you'll want to update the phase with the results of the simulation using ``sf.update_results()``, (assuming your StokesFlow algorithms was called ``sf``) which writes the pressure values back into the associated phase.
